Orange/widgets/unsupervised/owpca.py

```diff
@@ -125,6 +126,8 @@ def __init__(self):
         self.options_box = gui.vBox(self.controlArea, "Options")
         gui.checkBox(self.options_box, self, "normalize", "Normalize data",
                      callback=self._update_normalize)
+        self.center_box = gui.checkBox(self.options_box, self, "center", 
+            "Center data", callback=self._update_center)
```

Comment:
I'm not sure `Center data` is the right name for this checkbox. The problem as I see it, is that we also have `Normalize data` option above, which can be misleading since data normalization by default implicates data centering. If I select Normalize data and don't select Center data I would expect that the data is only scales, but this is not what happens.

 In essence this checkbox is selecting between PCA and TruncatedSVD. Should we maybe put them to a dropdown?